src/www/ui/template/browse_file.js.twig

```diff
@@ -60,12 +60,25 @@ function createDirlistTable() {
   };
 
   otable = $('#dirlist').dataTable(dirlistTableConfig);
+
 }
 
 $(document).ready(function () {
   createDirlistTable();
   $(document).tooltip();
   $("#lichistogram_length select").css('text-align','right');
-  $("#dirlist_length select").css('text-align','right');  
+  $("#dirlist_length select").css('text-align','right');
+
+  var i = 0;
+  $('#markIrrelevant').click(function () {
+    var currentArray = [];
+    $("input:checkbox[id=selectedForIrrelevant]:checked").each(function() {
+      getCurrentVal = $(this).val();
+      if(jQuery.inArray(getCurrentVal, currentArray) < 0 && getCurrentVal != ""){
+        currentArray[i++] = getCurrentVal;
```

Comment:
why not `currentArray.push(getCurrentVal)`? (this makes the variable `i` unnecessary)